Extension/src/githubAPI.ts

```diff
@@ -44,6 +44,9 @@ function getVsixDownloadUrl(build: Build, vsixName: string): string {
     return downloadUrl;
 }
 
+export function getOfflineDownloadUrl(): string {
```

Comment:
This could probably just be a const value instead of a function.  I suspect it could be used elsewhere in this file too though I don't know off the top of my head.

Reply:
I found it in main.ts being used in a log message (line 318). I can replace that one with the const value as well.